### ISSUE TITLE
ergoCub 1.1 S/N:001 – Screen is not functional

### ISSUE THREAD

**Issue by mebbaid** [robotology/icub-tech-support#1978]
### Robot Name 🤖

ergoCub 1.1 S/N:001

### Request/Failure description

I turned on the robot this morning and the screen is barely legible. 

### Detailed context

See attached picture

![Image](https://github.com/user-attachments/assets/341382a9-bc6a-489a-8417-eafdd21f244a)

### How does it affect you?

We are unable to monitor the status of the battery and the robot

**Comment by S-Dafarra** [comment]
Just to give some context, the screen broke on the SN002 robot, and we swapped the broken screen with the one of the 001 (this is also the reason why the screen is reversed).

We noticed that this issue happened when pressing the cover to insert the battery. Looking inside the cover we noticed that when pushed, it could have pressed on the corner of the screen, thus breaking the LCD. @AntonioConsilvio used the Dremel to remove some material from the edge close to the screen
![Image](https://github.com/user-attachments/assets/f5d86ead-91db-4d86-936f-7c4abe84c372)


cc @maggia80 @marcellosavoldi
